### ISSUE TITLE
Do you have the equipment?  Do you want me to test it?  Does it support decryption now? 

### ISSUE THREAD

**Issue by xxroot** [KatoTempest/motorola_pokerpatas#1]
Do you have the equipment?  Do you want me to test it?  Does it support decryption now?

**Comment by KatoTempest** [comment]
I am testing that the encryption works (at the moment it does not work, the recovery starts but cannot decrypt), at the moment I found some files related to that in the stock rom, the problem is that last night I applied one and the recvoery did not start, I want to suppose that I must simplify all that file with that setting

**Comment by xxroot** [comment]
The device tree I provided has been modified by enthusiastic netizens, I now provide a device tree can be mobilized.  As for the decryption function, if you have time, research it

**Comment by xxroot** [comment]
[E6 plus device tree](https://drive.google.com/file/d/1RoEkfkTqjoGChho3vZv81WCRnkV_fzFJ/view?usp=sharing)

**Comment by KatoTempest** [comment]
I will review those files later, although I did not fully understand what you wrote (I suppose you used translator or similar)

**Comment by xxroot** [comment]
Yes, I'm Chinese, I don't speak English, I use Google Translate. haha

**Comment by xxroot** [comment]
Yes, I'm Chinese, I don't speak English, I use Google Translate.

**Comment by xxroot** [comment]
The file I provided is working, but without decryption
